packages/office-ui-fabric-react/src/components/ResizeGroup/ResizeGroup.Props.ts

```diff
@@ -29,4 +31,13 @@ export interface IResizeGroupProps extends React.HTMLAttributes<ResizeGroup | HT
    * an infinite render loop.
   */
   onReduceData: (prevData: any) => any;
+
+  /**
+   * Function to be called when the measuring pass is complete and renderedData is shown on screen.
+   * It is only called when the data used to render does not match that of the last render.
```

Comment:
Are you suggesting always invoking the callback when there is a render and letting the consumer decide if they only care about new data?